packages/patternfly-react/src/components/HorizontalNav/HorizontalNavMenuItem.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { Dropdown, ListGroup, ListGroupItem } from '../../index';
+
+const HorizontalNavMenuItem = props => {
+  const {
+    children,
+    onItemClick,
+    title,
+    active,
+    dropdown,
+    submenu,
+    dropup,
+    pullLeft
+  } = props;
+
+  const dropdownClasses = classNames({
+    'dropdown-submenu': submenu,
+    'pull-left': pullLeft
+  });
+
+  return dropdown ? (
+    <Dropdown componentClass="li" className={dropdownClasses}>
+      <a bsRole="toggle">
+        {title}
+        {!submenu && <span className="caret" />}
+      </a>
```

Comment:
This should use a Dropdown.Toggle` with the componentClass set to "a", such as:
```
      <Dropdown.Toggle
        componentClass="a"
        bsStyle="link"
        noCaret={submenu}
      >
        {title}
      </Dropdown.Toggle>
```

Reply:
I tried but this add some extra style from bootstrap that break patternfly css, something similar happened in the infotip see https://github.com/patternfly/patternfly-react/blob/master/packages/patternfly-react/src/components/InfoTip/InfoTipToggle.js I'll change it and I'll report the problem

Reply:
With Dropdown.Toggle  classes are dropdown-toggle **btn btn-default**, and after set componentClass="a" to classes are the same and I can't override teh class to remove **btn** and **btn-default**

Reply:
Hi @jeff-phillips-18  Dos you check this?

Reply:
OK, we can leave it be then.